packages/forms-web-app/src/dynamic-forms/dynamic-components/listed-building-add-more/question.js

```diff
@@ -0,0 +1,61 @@
+const AddMoreQuestion = require('../add-more/question');
+
+const uuid = require('uuid');
+
+class ListedBuildingAddMoreQuestion extends AddMoreQuestion {
+	constructor({
+		title,
+		question,
+		fieldName,
+		viewFolder,
+		html,
+		validators,
+		label,
+		inputClasses = 'govuk-input--width-10'
+	}) {
+		super({
+			title,
+			viewFolder,
+			fieldName,
+			question,
+			html,
+			validators
+		});
+
+		this.inputClasses = inputClasses;
+		this.label = label;
+	}
+
+	/**
+	 * adds a uuid to save the listed building
+	 * @param {ExpressRequest} req
+	 * @returns
+	 */
+	async getDataToSave(req) {
+		return { addMoreId: uuid.v4(), value: { [this.fieldName]: req.body[this.fieldName] } };
+	}
+
+	/**
+	 *
+	 * @param {Object.<Any>} answer
+	 * @returns The formatted string to be presented in the UI
+	 */
+	format(answer) {
+		const identifier = answer[this.fieldName];
+		const grade = 'Grade';
+		const address = 'Address';
+		return [identifier, grade, address].join('<br>');
+	}
+
+	/**
+	 * adds custom info to view model
+	 */
+	prepQuestionForRendering(section, journey, customViewData) {
```

Comment:
We shouldn't need the prepQuestionForRendering in here as that is handled on the ListAddMoreQuestion class